core/storage/auth/gae_models_test.py

```diff
@@ -28,13 +28,7 @@
     from mypy_imports import base_models
     from mypy_imports import user_models
 
-(auth_models, base_models, user_models) = (
-    models.Registry.import_models([
-        models.NAMES.auth,
-        models.NAMES.base_model,
-        models.NAMES.user
-    ])
-)
```

Comment:
Why are you removing this?

Reply:
SIr I removed this because because my ci/circlei setup were failing.

Reply:
Done